src/python/bot/untrusted_runner/host.py

```diff
@@ -157,7 +158,12 @@ def wrapped(*args, **kwargs):
       except grpc.RpcError as e:
         # For timeouts, which aren't fatal errors, resurface the right
         # exception.
-        if 'TimeoutError' in str(e, encoding='utf-8', errors='ignore'):
```

Comment:
I tested the original fix here Python 3, but not Python 2 with newstr. It actually doesn't work in 2, so replacing it with something like the fix above. This is also just intended as a stopgap and will not work properly in Python 3, but we can likely remove it after converting.